tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_InnerVarNameConflict.ts

```diff
@@ -13,5 +13,5 @@ function /*[#|*/f/*|]*/(): Promise<string> {
 async function f(): Promise<string> {
     const resp = await fetch("https://typescriptlang.org");
     var blob = resp.blob().then(blob_1 => blob_1.byteOffset).catch(err => 'Error');
-    return blob_1.toString();
+    return blob_2.toString();
```

Comment:
This isn't right, `blob_2` was never declared.

Reply:
FWIW, the previous name of `blob_1` wasn't correct either. This is a different bug, but I can wrap it in here too so we leave it in a good state.

Reply:
Fixed.

Reply:
I would have guessed `blob_2` was of type `void` (based only on my intuition and not any knowledge of TS or promises).

---
In reply to: [217201225](https://github.com/Microsoft/TypeScript/pull/27031#discussion_r217201225) [](ancestors = 217201225)